README.md

```diff
@@ -338,6 +338,13 @@ SOCIAL_AUTH_TUNNISTAMO_OIDC_ENDPOINT = 'https://tunnistamo.example.com/'
 
 Note that `client ID` becomes `KEY` and `client secret` becomes `SECRET`.
 
+If you are using Active Directory login method in Tunnistamo and want users
+AD groups you should add the following setting:
```

Comment:
This could be a little more specific. This doesn't tell _where_ the user might want the AD groups to appear. Is it included in some token(s) or elsewhere? If it's a JWT, is there a dedicated claim name? What is the type of the returned value; is it an array of strings, a single string containing group names with some separator character or something else? A small example could answer many of these questions.

Reply:
It's a whole topic of its own. But yeah maybe I can write a quick note about that.

Reply:
Is there already some documentation elsewhere about this subject? If yes, there could be a link to that instead of repeating same info here. If there's no such documentation elsewhere, then this is probably the spot.

Reply:
I don't think to token contents are currently documented anywhere. Or how the mapping from AD-groups to Django groups is used.
So adding the info here is the best place currently.